app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide :title, t :Help %>
+
+<h1><%= t :Help %></h1>
+<p>
+  <%= link_to t(:help_1) %>
+  <a href="http://www.railstutorial.org/help"><%= link_to t(:help_2) %></a>
+  <%= link_to t(:help_3) %>
+  <a href="http://www.railstutorial.org/book"><%= link_to t(:help_4) %></a>.
```

Comment:
please read https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to
and study to use link_to